### DISCUSSION TITLE
Is it good idea to enable both apparmor and SELinux on host where CRI-O will be used?

### DISCUSSION THREAD

**Discussion by alpana17** [cri-o/cri-o#8044] [Q&A]
Does AppArmor and SELinux interfere or conflict if enabled?
Shall we just enable one?
If yes, how to decide, which one to enable?

**Comment by alpana17** [comment]
@saschagrunert please suggest

**Comment by saschagrunert** [comment]
Yes both are not supposed to work together: https://lwn.net/Articles/804906/

Depending on the distribution, you'd either like to choose SELinux (Fedora, CentOS, RHEL) or AppArmor (Ubuntu, openSUSE).

**Comment by alpana17** [comment]


**Reply by saschagrunert** [reply]
I assume in the documentation of the distribution :upside_down_face:

**Reply by alpana17** [reply]
Thank you
